docs/api.rst

```diff
@@ -1,4 +1,4 @@
-.. _sec_api:
+.. _sec-api:
 
```

Comment:
Why the change here? I changed the separators to be underscores a while back so they will autocomplete easily in vim.

Reply:
Oops. That's an error! Fixed